src/SFA.DAS.FAT.Web/Views/Courses/Courses.cshtml

```diff
@@ -54,7 +55,23 @@
                                                 @sector.Route
                                             </label>
                                         </div>
-                                    } 
+                                    }
+                                </div>
+                            </fieldset>
+                            <fieldset class="govuk-fieldset">
+                                <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+                                    Level
+                                </legend>
+                                <div class="govuk-checkboxes govuk-checkboxes--small">
+                                    @foreach (var level in Model.Levels)
+                                    {
+                                        <div class="govuk-checkboxes__item">
+                                            <input @(level.Selected ? "checked='checked" : "") class="govuk-checkboxes__input" id="route-@level.Code" name="levels" type="checkbox" value="@level.Code" />
+                                            <label class="govuk-label govuk-checkboxes__label" for="route-@level.Code">
+                                                Level @level.Code - @level.Name
```

Comment:
Can this be moved into the ViewModel with a test around it please, can then just be `@level.Title` which is the concat logic there.